planning/costmap_generator/nodes/costmap_generator/costmap_generator_node.cpp

```diff
@@ -276,7 +276,6 @@ void CostmapGenerator::onLaneletMapBin(
   lanelet::utils::conversion::fromBinMsg(*msg, lanelet_map_);
 
   if (use_wayarea_) {
-    loadRoadAreasFromLaneletMap(lanelet_map_, &area_points_);
```

Comment:
I think, if you remove this we can't support the case when lanes are not in free space areas like this. (Please regard the red areas are not free space areas.)

Therefore, the vehicle can't plan a trajectory after reaching the blue circle point.
![image](https://user-images.githubusercontent.com/31987104/170972431-64f2b097-1649-4076-9bf6-076aa16dba3d.png)

Reply:
I did take this case into consideration but I think the lanes and the parking lot are meant to be overlapped as the comment  of "getLinkedParkingLot" indicates. 
https://github.com/autowarefoundation/autoware.universe/blob/main/map/lanelet2_extension/lib/query.cpp#L437
Not sure about that, if it is not the convention, maybe some guidelines for map producing should be provided.

Besides, it seems no departure strategy is implemented for parking scenario so far. So I think the car can not reach the blue point from a parking spot.

Reply:
@NorahXiong Thank you for your comment!

> I did take this case into consideration but I think the lanes and the parking lot are meant to be overlapped as the comment of "getLinkedParkingLot" indicates.

I see. Let me ask other engineers about the map spec. :bow: 
@mitsudome-r @xmfcx @yukkysaito Is it reasonable to assume that lanes are overlapped with free spaces in a parking lot?

> maybe some guidelines for map producing should be provided.

Yes, it is really desired. But there is no good one, we're sorry about that. :cry: 

> Besides, it seems no departure strategy is implemented for parking scenario so far. So I think the car can not reach the blue point from a parking spot.

In this case, I think the vehicle should start with the lane driving scenario.
(If not, I guess it's a bug.)